hasher-matcher-actioner/hmalib/lambdas/aws_secrets.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import boto3
+import base64
+import typing as t
+import os
+import ast
+from botocore.exceptions import ClientError
+
+from threatexchange.api import ThreatExchangeAPI
+
+AWS_REGION = os.environ["AWS_REGION"]
+
+class AWSSecrets():
+    """
+    A class for reading secrets stored in aws for interacting with ThreatExchange
+    """
+    def te_api_data(self) -> "ThreatExchangeAPI" :
+        secret_name = "threatexchange/api_keys"
+        secret_str = get_str_secret(secret_name)
+        return ThreatExchangeAPI(secret_str)
+
+def get_bin_secret(secret_name: str) -> t.BinaryIO:
```

Comment:
I think this returns a literal `bytes` object rather than a BinaryIO, which is a different interface